schema/pnts.featureTable.schema.json

```diff
@@ -40,6 +43,9 @@
             },
             "CONSTANT_RGBA" : {
                 "$ref" : "featureTable.schema.json#/definitions/globalPropertyCartesian3"
+            },
+            "BATCH_LENGTH" : {
```

Comment:
Okay, should we add it to `i3dm.featureTable` as well? I think that can have a similar use case: where multiple instances are part of the same batch.